obs/obs-api.yaml

```
@@ -156,6 +156,33 @@ paths:
                 format: binary
         default:
           $ref: "#/components/responses/ErrorResponse"
+  /files/{id}/internal:
+    get:
+      tags:
+        - Files
+      summary: Get contnet refernce data of the file
+      description: Use this endpoint to get the contnet reference data of the file on the S3.
```

```suggestion
      description: Use this endpoint to get the content reference data of the file on the S3.
```
still not understandable. what is "content reference data"?